### ISSUE TITLE
confusion with the code which makes "shuffle=False" in trainer.py

### ISSUE THREAD

**Issue by MengHao666** [XiaohangZhan/deocclusion#42]
Hi, I am deeply confused about make **shuffle=False** in train loader. Is there any special reason for this?https://github.com/XiaohangZhan/deocclusion/blob/ac543f9a54f4cb8baf0774bdb5f8638eadf8b57c/trainer.py#L95
```
self.train_loader = DataLoader(train_dataset,
                                           batch_size=args.data['batch_size'],
                                           shuffle=False,
                                           num_workers=args.data['workers'],
                                           pin_memory=False,
                                           sampler=train_sampler)
```

**Comment by XiaohangZhan** [comment]
shuffle should be False when given sampler. The data have already been shuffled in the sampler.

**Comment by MengHao666** [comment]
> data have already been shuffled in the sampler.

Got it! Thanks for helpful reply.
